catanatron_compiled/catanatron_compiled/src/catanatron_compiled.cc

```diff
@@ -0,0 +1,44 @@
+#include "graph.h"
+
+#include <pybind11/pybind11.h>
+
+namespace catanatron {
+
+// ----------------
+// Python interface
+// ----------------
+
+PYBIND11_MODULE(catanatron_compiled, m) {
+  m.doc() = "Compiled Catanatron features";
+
+  // Add Graph Module
+  py::module_ graph_module = m.def_submodule("graph", "Compiled Graph features");
```

Comment:
make these docs more descriptive, please (what are the compiled Graph features?)